.circleci/config.yml

```diff
@@ -333,47 +334,322 @@ version: 2.1
 workflows:
     cnfpt:
         jobs:
-            - no-change:
```

Comment:
Not sure how all these jobs changes are relevant to translations? Maybe this deserves a mention in commit messages at least if it is intended